### ISSUE TITLE
Error: media query expression must begin with '('

### ISSUE THREAD

**Issue by anthony-robin** [sass/sassc-rails#57]
Not sure if this related to sassc library but when I used foundation 6 gem into my rails project, I got this compiling error:
``` shell
Error: media query expression must begin with '('
on line 138 of ../../../../.rvm/gems/ruby-2.2.2/gems/foundation-rails 6.1.1.3/vendor/assets/scss/util/_breakpoint.scss
```

The file where is issue happend is:
https://github.com/zurb/foundation-rails/blob/master/vendor/assets/scss/util/_breakpoint.scss#L138

**Comment by cgs** [comment]
Having the same issue, except it's pointing to breakpoint-sass:

`Error: media query expression must begin with '('
        on line 65 of vendor/bundle/gems/rails-assets-breakpoint-sass-2.6.1/app/assets/stylesheets/breakpoint-sass/_breakpoint.scss`

**Comment by mikemerritt** [comment]
I can also confirm this issue. Compilation with sass-rails works fine but sassc-rails will not compile due to this error. Seems to be an issue with libsass 3.3.3 that they are aware of and should be fixed when 3.3.4 is released. 

A temporary workaround is to add `gem 'sassc', '1.8.2'` to your gemfile so that it reverts to libsass 3.3.2. You'll need to run `bundle update sassc` to downgrade to 1.8.2.

**Comment by bolandrm** [comment]
Hello, I've released sassc 1.8.5.  Could you please give that a try?

**Comment by pedropereira** [comment]
@bolandrm That fixed it for me, thanks.
